Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -12813,6 +12813,83 @@ final class WorkspaceTests: XCTestCase {
             ["did load manifest for registry package: org.baz (identity: org.baz)"]
         )
     }
+    
+    func testTransitiveResolutionFromRegistryWithByNameReferencesWhenSourceControlToRegistryDependencyTransformationIsSwizzle() async throws {
```

Comment:
This one is on the opposite end of the spectrum, needs a more concise name. If there are any crucial details omitted in the name, those can be included in function's doc comment.

Reply:
Updated to `testTransitiveResolutionFromRegistryWithByNameDependencies`. Let me know if that works for you.